{"Functions": {"InvalidCredentialsException()": {"Returns": [], "Parameters": [], "Throws": [], "SeeAlso": [], "Permissions": [], "Description": "Creates a new InvalidCredentialsException with a", "history": "Added in API level 1", "FullName": "public InvalidCredentialsException ()"}, "InvalidCredentialsException(java.lang.String, java.lang.Throwable)": {"Returns": [], "Parameters": [["@B_java#lang#String_E@", "the exception detail message"], ["@B_java#lang#Throwable_E@", "the Throwable that caused this exception, or null if the cause is unavailable, unknown, or not a Throwable"]], "Throws": [], "SeeAlso": [], "Permissions": [], "Description": "Creates a new InvalidCredentialsException with the specified detail message and cause.", "history": "Added in API level 1", "FullName": "public InvalidCredentialsException (String message, Throwable cause)"}, "InvalidCredentialsException(java.lang.String)": {"Returns": [], "Parameters": [["@B_java#lang#String_E@", "the exception detail message"]], "Throws": [], "SeeAlso": [], "Permissions": [], "Description": "Creates a new InvalidCredentialsException with the specified message.", "history": "Added in API level 1", "FullName": "public InvalidCredentialsException (String message)"}}, "Inheritance": ["@B_java#lang#Throwable_E@", "@B_java#lang#Exception_E@", "@B_org#apache#http#HttpException_E@", "@B_org#apache#http#ProtocolException_E@", "@B_org#apache#http#auth#AuthenticationException_E@"], "ClassName": "org.apache.http.auth.InvalidCredentialsException", "ClassDesc": "Please use @B_java#net#URL#openConnection_E@ instead. Please visit Authentication credentials required to respond to a authentication challenge are invalid"}